### PR TITLE
Menu restyling and organization

### PR DESCRIPTION
**Pull request by megalodonatron**
Fixes issue #570

screen shot 2014-01-09 at 1 02 37 pm
screen shot 2014-01-09 at 11 40 56 am

### CODE REVIEW

File: www/americangut/index.psp [hunk 1]
```diff
@@ -217,7 +218,8 @@ ag_data_access = data_access_factory(ServerConfig.data_access_type, 'american_gu
 # if database connection was unsuccessful, this is the first place it would
 # manifest
 try:
-    results = ag_data_access.getMapMarkers()
+    # results = ag_data_access.getMapMarkers()
+    results = []
```
Comment (adamrp) [inline www/americangut/index.psp]: we have to remember NOT TO MERGE THIS LIVE with this change in place.

Comment (megalodonatron) [inline www/americangut/index.psp]: I can take it out now, it's just that nothing will work for testing on webdev with it in there

Comment (adamrp) [inline www/americangut/index.psp]: It will work on webdev, just won't work from virtual box

**Comment by megalodonatron** [comment]
Also this should get merged before my other pull req (#571)  because I'll need to put the "log sample" item in a different place once this is merged

**Comment by adamrp** [comment]
Thanks, Meg.  I'm going to deploy this to webdev now so people can check it out, and I'll add any comments I have in a few minutes.

**Comment by adamrp** [comment]
Uh oh -- I guess webdev has now gone over its google API limit, too!  Note that this will not continue to happen after this week, since this is related to the addition of the "elevation" column to the ag_login table, so after all samples are updated from the API, we won't run into this issue any more.

At any rate, I'll make the bandaid change on webdev; I don't think we want to have that kind of thing in the pull request itself.

**Comment by teravest** [comment]
Adam,

Have you tried changing the ip address of your virtual machine to get
around it?  Just a thought to try and speed this up.  We dont want to synch
that code to prod until we get this worked out.  Also, how are we gonna get
all those values updated on production?

**Comment by adamrp** [comment]
- I think the link to the FAQ should not open in a new tab (others' opinions?) and the same "participant resources" menu could be added to FAQ.psp
- Mousing over the "International Shipping" option highlights both the english and the spanish instructions (is this intended?)
- I think the spanish instructions should have more text than just "(es)"

I'm going to add these two points to another issue (while you're working on menus, figured you might want to know about them haha):
- Change the "Join the project" link to open in a new tab
- The menu on addendum.psp has different styling than the other menus

**Comment by adamrp** [comment]
@teravest, I have fundamentally changed the geocoding stuff so that we manually choose when to geocode new addresses, and how many to do at a time.  See #572

**Comment by megalodonatron** [comment]
I fixed all of the issues that @adamrp raised including "join the project" opening in a new tab.

The addendum page is weird to me, it only shows up when you're logged in but it's written as though it should be shown even when you're not logged in.

If it should only be shown when logged in then it should be formed like portal.psp not index.psp.

**Comment by ElDeveloper** [comment]
> The addendum page is weird to me, it only shows up when you're logged in but it's written as though it should be shown even when you're not logged in.

This is something that I brought up in some e-mail discussion, but nobody really had strong feelings about it. I in particular don't, and kind of think that it's only participants with their password that should be able to read this information, though then again I don't really have a strong feeling about it. I'm ok with changing it or leaving it as it is.

**Comment by adamrp** [comment]
It kind of makes sense to me to make this information available to anyone
who visits the site.  Who knows, some curious visitor might read it and
decide to sign up!

**Comment by ElDeveloper** [comment]
+1

**Comment by megalodonatron** [comment]
Okay well that should be in a different issue/pull req, so if people could OK and merge this one that would be greeeeeeeeeeeeaaaaaaaaaaat....... 
image

**Comment by ElDeveloper** [comment]
:+1:

**Pull request merged**